src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -1196,6 +1209,20 @@ private static void GetMatchingNamespaceOrTypes(INamespaceOrTypeSymbol container
                         results.Add(symbol);
                     }
                 }
+
+                GetMatchingExtensions(container, memberName, results); // Note: the arity is already in the content-based extension grouping name
```

Comment:
>// Note: the arity is already in the content-based extension grouping name [](http://example.com/codeflow?start=71&length=74)

I do not think we can rely on this for imported symbols, other tools are free to generate names differently.